mml-startup Error: Unable to initialize main class megameklab.MegaMekLab

### Environment
What version of MekLab does your issue apply to?  MekLab standalone
What operating system are you using?  MacOS 10.15 and Gentoo Linux
What java version are you using? 
```
MacBook-Pro:megameklab-0.49.13 me262$ java --version
java 17.0.5 2022-10-18 LTS
Java(TM) SE Runtime Environment (build 17.0.5+9-LTS-191)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.5+9-LTS-191, mixed mode, sharing)
```
```
me262@cliffjumper ~/Downloads/megameklab-0.49.13 $ java --version
openjdk 17.0.6 2023-01-17
OpenJDK Runtime Environment Temurin-17.0.6+10 (build 17.0.6+10)
OpenJDK 64-Bit Server VM Temurin-17.0.6+10 (build 17.0.6+10, mixed mode, sharing)
```

### Description
MegaMekLab standalone cannot startup, and the full package must be downloaded in order to start. Previously listed as bug #1218 .

Error message: 
```
MacBook-Pro:megameklab-0.49.13 me262$ ./mml-startup 
Error: Unable to initialize main class megameklab.MegaMekLab
Caused by: java.lang.NoClassDefFoundError: megamek/client/ui/preferences/SuitePreferences
```
```
me262@cliffjumper ~/Downloads/megameklab-0.49.13 $ ./mml-startup
Error: Unable to initialize main class megameklab.MegaMekLab
Caused by: java.lang.NoClassDefFoundError: megamek/client/ui/preferences/SuitePreferences
```


### Files
No logfiles are produced.

--- a/megameklab/build.gradle
+++ b/megameklab/build.gradle
@@ -110,8 +110,8 @@ jar {
     archiveFileName = "MegaMekLab.jar"
     manifest {
         attributes "Main-Class" : mainClassName
-        attributes "Class-Path" : "${mmJar.archiveFileName.get()} " + (project.sourceSets.main.runtimeClasspath.files
-                .findAll { it.name.endsWith(".jar") }
+        attributes "Class-Path" : "${lib}/${mmJar.archiveFileName.get()} " + (project.sourceSets.main.runtimeClasspath.files
+                .findAll { it.name.endsWith(".jar") && !it.name.toLowerCase().startsWith("megamek") }
                 .collect { "${lib}/${it.name}" }.join(' '))
         attributes "Add-Opens" : 'java.base/java.util java.base/java.util.concurrent'
         attributes "Build-Date" : LocalDateTime.now()

